packages/pipeline/src/parsers/ddex_orders/index.ts

```diff
@@ -0,0 +1,73 @@
+import { BigNumber } from '@0x/utils';
+import * as R from 'ramda';
+
+import { DdexMarket, DdexOrder, DdexOrderbook } from '../../data_sources/ddex';
+import { TokenOrderbookSnapshot as TokenOrder } from '../../entities';
+import { OrderType } from '../../types';
+
+/**
+ * Marque function of this file.
+ * 1) Takes in orders from an orderbook,
+ * 2) Aggregates the orders based on price point,
+ * 3) For each aggregated order, forms a TokenOrder entity with market data and
+ * other information attached.
+ * @param ddexOrderbook A raw orderbook that we pull from the Ddex API.
+ * @param ddexMarket An object containing market data also directly from the API.
+ * @param retrievalTimestamp Time at which the orders for the market were pulled.
+ * @param source The exchange where these orders are placed. In this case 'ddex'.
+ */
+export function parseDdexOrders(ddexOrderbook: DdexOrderbook, ddexMarket: DdexMarket, retrievalTimestamp: BigNumber, source: string): TokenOrder[] {
+    const aggregatedBids = aggregateOrders(ddexOrderbook.bids);
+    const aggregatedAsks = aggregateOrders(ddexOrderbook.asks);
+    // Might want to shift 'bid', 'ask' enums to 0x/types and retrieve from there.
+    const parsedBids = R.map(parseDdexOrder(ddexMarket, retrievalTimestamp, 'bid', source), aggregatedBids) ;
```

Comment:
nit: There's a space before the semicolon here.

This sort of formatting issue is best suited for robots, not humans. Were you able to set up [Prettier and TSLint](https://github.com/0xProject/0x-monorepo/blob/development/CONTRIBUTING.md#development-tooling)? They can automatically format your code and save you a ton of time.

Reply:
Oh darn. I have TSlint setup, but it didn't pick out this one. Strange eh?